ivadomed/main.py

```diff
@@ -308,6 +308,10 @@ def run_command(context, n_gif=0, thr_increment=None, resume_training=False):
     """
     command = copy.deepcopy(context["command"])
     path_output = set_output_path(context)
+    log_file = context['log_file']
```

Comment:
Sorry, I do not understand the difference of this line vs [this one](https://github.com/ivadomed/ivadomed/blob/34551756dd6cc3cbec7800a691683096d6a7305b/ivadomed/main.py#L480).

Reply:
Also, would be good to have the log file saved within the log_directory, is it the case?

Reply:
no, but I'll change it now

Reply:
> Sorry, I do not understand the difference of this line vs [this one](https://github.com/ivadomed/ivadomed/blob/34551756dd6cc3cbec7800a691683096d6a7305b/ivadomed/main.py#L480).

the one you mentioned is version_info.log, this one is the log for all logger.info/warning/etc